### PR TITLE
Companies table

### PR DESCRIPTION
**Pull request by cyborch**
### What

Add companies table, including:

 * All migrations needed to reflect the columns in that table.
 * The new permissions and relationships in Hasura.
 * Styling of the table.

### UI

Screenshot 2023-04-26 at 16 21 05

### CODE REVIEW

File: front/src/interfaces/company.interface.test.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      created_at: '',
+      account_owner: {
+        id: '',
+        email: '',
+        displayName: '',
+      },
+      employees: 0,
+      address: '',
+    });
+    expect(company.name).toBe('ACME');
```
Comment (charlesBochet) [inline front/src/interfaces/company.interface.test.ts:18]: shouldn't we assert the whole object?

File: front/src/interfaces/company.interface.test.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { mapGqlCompany, mapCompany } from './company.interface';
```
Comment (charlesBochet) [inline front/src/interfaces/company.interface.test.ts:1]: Nice to add tests!

File: front/src/interfaces/company.interface.test.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      opportunities: [],
+      accountOwner: {
+        id: '',
+        email: '',
+        first_name: '',
+        last_name: '',
+      },
+      creationDate: new Date(),
+    });
+    expect(company.name).toBe('ACME');
```
Comment (charlesBochet) [inline front/src/interfaces/company.interface.test.ts:37]: same here, shouldn't we assert the whole company object with all its fields?

File: front/src/interfaces/company.interface.ts [hunk 4]
```diff
@@ -1,5 +1,59 @@
+import { User } from './user.interface';
+
+export interface Opportunity {
```
Comment (charlesBochet) [inline front/src/interfaces/company.interface.ts:3]: I would move this into opportunity.interface.ts for consistency

**Review by charlesBochet — APPROVED** [review]
Nice!

File: front/src/interfaces/company.interface.ts (hunk 4 above)
Comment (cyborch) [reply front/src/interfaces/company.interface.ts:3]: I saw a separate task about adding opportunities in Linear, so I added this comment in the commit msg:

```
Opportunity is expected to be replace by actual opportunity in a separate PR.
```

File: front/src/interfaces/company.interface.test.ts (hunk 1 above)
Comment (cyborch) [reply front/src/interfaces/company.interface.test.ts:18]: We can definitely start doing that 😄

**Pull request merged**